draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1041,20 +1080,21 @@ pass the following result on to the application for further processing:
 
 <{{examples/complex_ekyc/verified_contents.json}}
 
-## Example 4 - W3C Verifiable Credentials Data Model (work in progress)
+## Example 4a - W3C Verifiable Credentials Data Model v2.0, not using JSON-LD
```

Comment:
I suggest to state "using JSON payload" or "JSON claim set" that reads much more positive than "not ..."

Reply:
JSON-LD is JSON, too. We say `without using JSON-LD` in OID4VP spec. I want to be explicit that it is not JSON-LD

Reply:
I agree with @tlodderstedt ... The example and text should be what makes it explicit.

perhaps "W3C Verifiable Credentials as Native JSON Web Tokens" ?

Reply:
keeping the language for now after discussion with other editors. please file an issue if you feel strongly about it.